sdk/storage/Azure.Storage.Blobs.AspNetCore.DataProtection/src/Azure.Storage.Blobs.AspNetCore.DataProtection.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <Description>Microsoft Azure Blob storage support as key store.</Description>
```

Comment:
Can we include a link to the ASP.NET documentation on key stores?  Maybe https://docs.microsoft.com/en-us/aspnet/core/security/data-protection/implementation/key-storage-providers would be best here.